al-jazari/roster.json

```diff
@@ -77,6 +77,10 @@
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "video",
+          "url": "https://youtu.be/_m7hG40JQZI"
```

Comment:
Can you please change the video object to
  {
          "type": "video",
          "url": "_m7hG40JQZI"
   }